Safety/Src/safety_controller.cpp

```diff
@@ -0,0 +1,67 @@
+#include "safety_controller.hpp"
+#include "PWM.hpp"
+#include "stdlib.h"
+#include "stm32f0xx_hal_iwdg.h"
```

Comment:
I don't think we should be including any HAL files directly into .cpp files. I remember Anthony mentioning something about this to me.

Reply:
yeee you right, that was leftover code